tests/ErrorMessages.hs

```diff
@@ -141,7 +141,7 @@ outputGeneric choice = concat
       (select
         thSomeTypeParseJSONRejectUnknownFields
         gSomeTypeParseJSONRejectUnknownFields)
-      [ "{\"tag\": \"record\", \"testOne\": 1.0, \"testZero\": 1}"
+      [ "{\"tag\": \"record\", \"testone\": 1.0, \"testZero\": 1}"
```

Comment:
`optsRejectUnknownFields` actually had the `fieldLabelModifier = map toLower` set all the time. Hence this test input should have rejected the `testOne` originally and prior to my bugfix did not. 

I chose to not change the expectations in the golden test but instead made the test input comply with the expectation.